app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,53 @@
+<%if@recipes.empty?%>
+<h1 class="text-important font-important text-center">The list is empty</h1>
+<%else%>
+<ul class="flex flex-col gap-5">
+  <% @recipes.each do |recipe|%>
+  <li>
+    <a href="/users/<%= @user.id%>/recipes/<%=recipe.id%>" class="grid grid-cols-1 sm:grid-cols-3 p-3 bg-gradient-to-r hover:from-orange-400 hover:to-yellow-500 min-h-[130px] text-start hover:shadow-[10px_9px_#00a195] border-b border-black cursor-pointer rounded-md">
+      <div class="col-span-1 relative flex flex-col text-center">
+        <h2 class="font-important text-important text-center m-auto"><%= recipe.name %></h2>
+        <%= button_to "Remove", user_recipe_path(current_user,recipe),class: "button underline hover:text-red-600", method: :delete %>
+
+        </div>
+      <p class="col-span-2 py-4 px-2"><%= recipe.description %></p>
+    </a>
+    </li>
+  <%end%>
+</ul>
+
+<%end%>
+
+
+<script src="https://cdn.jsdelivr.net/npm/axios@1.1.2/dist/axios.min.js" crossorigin="anonymous"></script>
+<script>
+  const deleteButtons = document.querySelectorAll('.button');
+  deleteButtons.forEach(button => {
+      button.addEventListener('click', async (e) => {
+        const confirm = window.confirm("Are you sure you want to delete this recipe?")
+        if(!confirm){
+            return;
+        }
+          e.preventDefault();
+          // disable the button
+          button.disabled = true;
+         const response = await axios.delete(button.parentElement.action, {
+          headers: {
+              'X-CSRF-Token': document.querySelector('meta[name="csrf-token"]').getAttribute('content'),
+              'Content-Type':  'application/json',
+         } })
+         if(response.status === 200) {
+             button.parentElement.parentElement.parentElement.remove()
+             const notice = document.querySelector('.notice')
+                notice.innerHTML = "Delete successful"
+         } else{
+           const error = document.querySelector('.alert')
+           // remove the disabled attribute
+              button.removeAttribute('disabled')
+          error.innerHTML = "Delete failed"
+         }
+
+
+      })
+  })
```

Comment:
Please correct the above code 
If a user clicks on the cancel button, they should stay on the same page that shows the list of items, instead of being directed to the individual item's show page.